src/csv_to_confluence.py

```diff
@@ -783,7 +789,29 @@ def create_summary_table_row_from_confluence_page(self, commonname: str, conflue
         todays_imports = self.__get_sum_of_two_table_data_elements(template, 'daily_imported', 'daily_updated')
         todays_errors = self.__get_sum_of_two_table_data_elements(template, 'daily_invalid', 'daily_failed')
         row = self.__creator.create_html_element('tr')
-        row.extend([node, interface, last_check, status, todays_imports, todays_errors, todays_error_rate, last_weeks_error_rate])
+        row.extend([node, interface, last_check, status, todays_imports, todays_errors, todays_error_rate,
+                    last_weeks_error_rate])
+        return row
+
+    def create_histogram_table_row(self, row_element: bs4, page_name: str, file_path: str) -> Tag:
```

Comment:
The histogram should be above the table itself. Not in a row....

Reply:
I wrapped the histogram and table into a div, and updated the confluence page with them. The problem was that if i updated the page separately it removed the previous update so i needed to upload the histogram and table at the same time.

Reply:
Do not update the Confluence Page directly but only the Template, that will be uploaded